src/python/opentx.py

```diff
@@ -31,48 +16,3 @@ def gen_elrs(source, target, env):
     sys.stdout.write("=====================================================================================================================================\n")
     sys.stdout.write("\n")
     sys.stdout.flush()
-
-
-def gen_frsky(source, target, env):
```

Comment:
`.frk` file generation removed since `.elrs` is supported by OpenTX now. STK500 protocol is better and avoid misunderstanding with file ends (`.frk` points only to bootloader now)